components/PlantList.tsx

```diff
@@ -2,21 +2,79 @@ import React, { useEffect, useState } from 'react';
 import { getPlantSeasonality } from '@/api/supabase/queries/plantSeasonality';
 import { Plant } from '@/types/schema';
 
-export const PlantList = () => {
+interface PlantListProps {
+  harvest_season: string;
+  planting_type: string;
+  growing_season: string;
+}
+
+export const PlantList = (props: PlantListProps) => {
   const [plants, setPlants] = useState<Plant[]>([]);
+  const growingSeasonToMonth = new Map<string, string[]>([
+    ['Spring', ['MARCH', 'APRIL', 'MAY']],
+    ['Summer', ['JUNE', 'JULY', 'AUGUST']],
+    ['Fall', ['SEPTEMBER', 'OCTOBER', 'NOVEMBER']],
+    ['Winter', ['DECEMBER', 'JANUARY', 'FEBRUARY']],
+  ]);
 
   useEffect(() => {
     const fetchPlantSeasonality = async () => {
-      const plantList = await getPlantSeasonality('Tennessee');
+      // gets plants in Tennessee by default
+      const plantList = await getPlantSeasonality('TENNESSEE');
```

Comment:
`getPlantSeasonality` has been refactored into `getAllPlants` on main, so we'll need to rebase and address the conflict here!